packages/multichain/src/scope/authorization.test.ts

```diff
@@ -0,0 +1,91 @@
+import { validateAndNormalizeScopes } from './authorization';
+import * as Transform from './transform';
+import type { ExternalScopeObject } from './types';
+import * as Validation from './validation';
+
+jest.mock('./validation', () => ({
+  getValidScopes: jest.fn(),
+}));
+const MockValidation = jest.mocked(Validation);
+
+jest.mock('./transform', () => ({
+  normalizeAndMergeScopes: jest.fn(),
+}));
+const MockTransform = jest.mocked(Transform);
+
+const validScopeObject: ExternalScopeObject = {
+  methods: [],
+  notifications: [],
+};
+
+describe('Scope Authorization', () => {
+  describe('validateAndNormalizeScopes', () => {
+    it('validates the scopes', () => {
+      try {
```

Comment:
This shouldn't throw 🤔 Why would this be in a try block?

Reply:
fixed here: https://github.com/MetaMask/core/pull/4784/commits/858f76460d5b58de74b56cabb859eba204466054